corpus/data_augmentation.py

```diff
@@ -236,3 +243,167 @@ def run(self):
 
         with open(str(self.out_segment_file), "w") as segments_outfile:
             segments_outfile.writelines(segment_file_names)
+
+
+class AddImpulseResponseJob(Job):
+    """
+    Add impulse responses to each recording in the corpus with a given probability to be affected.
+    This is done by convolving the audio with an impulse response to simulate other acoustic conditions.
+    Supports .wav files
+
+    This job is dependent on the audiomentations library.
+    """
+
+    def __init__(
+        self, corpus_file, new_corpus_name, ir_path, augment_prob, time_rqmt=12
+    ):
+        """
+
+        :param Path corpus_file: Bliss corpus with wav files
+        :param str new_corpus_name: name of the new corpus
+        :param str ir_path: path to the directory containing impulse responses
```

Comment:
A question here: it seems the ir_path will contain multiple IRs, so will only one be picked randomly or will all be applied?

Reply:
One will be picked randomly for each audio.